ldm/invoke/generator/base.py

```diff
@@ -63,7 +63,7 @@ def generate(self,prompt,init_image,width,height,sampler, iterations=1,seed=None
             **kwargs
         )
         results             = []
-        seed                = seed if seed is not None else self.new_seed()
+        seed                = seed if seed is not None and seed > 0 else self.new_seed()
```

Comment:
You're right. I'll change this to >= 0 so that the only negatives or none trigger a new seed.

Reply:
Doesn't negative seed look back in history to reuse a previous seed? I thought I read that in docs somewhere.

Reply:
Yes, it does, and I am now special-casing such that the postprocessing operations will receive the actual negative value. I guess this will help out in other situations in which the user wants to replace the seed of a previously-generated image. I agree this is ugly and will need to be ripped out at some point, but it's handy to be able to say "I didn't like the outpaint you generated, please try another". 

In the post-node CLI, I will use a syntax more like -S[-1] to pick previous values. This will apply to all arguments. However, since I'm using typed arguments from argparse, it is major surgery to do it now.

Reply:
I need to output seed from generations, but otherwise that's already possible with the node CLI 😊